picard/util/remotecommands.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, method_name, help_text=None, help_args=None):
     "LOOKUP_CD": RemoteCommand(
         "handle_command_lookup_cd",
         help_text="Read CD from the selected drive and lookup on MusicBrainz. "
-        "Without argument, it defaults to the first (alphabetically) available disc drive",
+        "Without argument, it defaults to the first (alphabetically) available disc drive.",
```

Comment:
This doesn't look like related to PICARD-2615.

Reply:
Strictly speaking it isn't, but I thought I would include the change here rather than create a separate PR just to add a period to one description to make it consistent with all the others.  I'll remove it from this PR and create a new one just for addinding the period.

Reply:
I just added https://github.com/metabrainz/picard/pull/2193 to cover the change to the `LOOKUP_CD` command description.